R/find_groups.R

```diff
@@ -0,0 +1,70 @@
+#' Find meetup groups using text-based search
+#'
+#' @param text string used to filter groups
+#' @param radius can be either "global" (default) or distance in miles in the
+#' range 0-100.
+#' @template api_key
+#'
+#' @return A tibble with the following columns:
```

Comment:
Do you think we should extract all of these columns, or just provide a vignette that shows how to pull extra columns out of the `resource` column? I'm a bit hesitant to give so much back since it becomes a bit cumbersome.

Reply:
If having it readily available saves our users some time, I think it's worth it, especially because a lot of people don't read vignettes and we want to make it accessible to beginners.  I went through the list and added all the ones I thought might be useful for a data analysis.  

Though if you feel strongly about it, here's the columns that I think are lower priority:
- `organizer_bio`
- `organizer_photo_link`
- `group_photo_link`
- `category_id`

I think the `description` column is valuable, but it also has the most text in it, so if we are looking to trim down to stuff that looks more like metadata, we could get rid of that as well.  

I'm open to either option; let me know what you think.  Either way I like your idea of adding how to add columns to a vignette.  We could to extend @cvitolo's vignette to demonstrate how to use `purr::map_chr()` to add a column and mention the other data types, `purr::map_dbl()`, `purr::map_int()`.

Reply:
👌 I'm fine with that if you think the average beginner would need these columns, then leaving them makes sense (I haven't done many analyses with the meetup API, so admittedly I don't know what columns I _would_ like if I were doing the analysis). Do you want to add to the vignette for this PR, or should I open an issue for a subsequent?

Reply:
I just made some additional changes to the output of `find_groups()`.  We can do the vignette and other changes in a follow-up PR.